### PR TITLE
Enhance documentation

### PR DESCRIPTION
**Pull request by rongzhus**

### CODE REVIEW

**Review by rui1234 — COMMENTED** [review]
I would suggest to move the content of synctl-cli.md into the README.md, so users can glimpse the cli usage and command list, they can also click the command link to see more if need.

**Comment by rui1234** [comment]
1. Synthetic CLI Documentation should be removed in the Table of Contents
2.  Command "synctl get pop-size - Estimate size of Instana hosted Synthetic PoP." should be "synctl get pop-size - Estimate size of Self-hosted PoP."
3. "Features" section:  add new feature "Estimate PoP size and cost", "Support multiple configurations of backend server." can be put as the first item

**Review by rui1234 — APPROVED** [review]

**Comment by rui1234** [comment]
in the https://github.com/instana/synthetic-synctl/blob/enhance-docs/docs/synctl-get-size.md doc, "Estimate size of Instana hosted Synthetic PoP." should be "Estimate size of Self-hosted PoP."

File: docs/synctl-get-size.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Size the PoP hardware configuration of Self-hosted PoP.
+
+## Syntax
+```
+synctl get pop-size|size
+```
+
+## Examples
+Estimate size of
```
Comment (swethalohith2) [inline docs/synctl-get-size.md:11]: incomplete sentence here

**Pull request merged**